README.md

```diff
@@ -355,8 +355,8 @@ Parameters of the driver script, `abyss-pe`
  * `b`: maximum length of a bubble (bp) [`10000`]
  * `c`: minimum mean k-mer coverage of a unitig [`sqrt(median)`]
  * `d`: allowable error of a distance estimate (bp) [`6`]
- * `e`: minimum erosion k-mer coverage [`sqrt(median)`]
- * `E`: minimum erosion k-mer coverage per strand [`1`]
+ * `e`: minimum erosion k-mer coverage [`round(sqrt(median))`]
+ * `E`: minimum erosion k-mer coverage per strand
```

Comment:
Ah, yes. Would you be up for documenting that as well?
`minimum erosion k-mer coverage per strand [1 if c > 2 else 0]`